assets/src/pages/room/components/StreamPlayer/UnpinnedTilesSection.tsx

```diff
@@ -48,12 +49,19 @@ const UnpinnedTilesSection: FC<Props> = ({
 
   const tileSize = tileConfigs.length >= 7 ? "M" : "L";
 
+  const getUpperLeftIcon = (config: MediaPlayerTileConfig) => {
+    if (config.typeName === "local" || config.typeName === "remote") {
+      if (showDisabledIcon(config.audio)) return <DisabledMicIcon isLoading={isLoading(config.audio)} />;
+      if (config.isSpeaking) return <SoundIcon />;
+    }
+  };
+
```

Comment:
I don't like an implicit return of undefined. Based on our previous commits, I think we should use null instead of undefined. What is more, we should allow components to declare props as `sth?: T | null`. 

I don't like nested `ifs` either. You could extract it outside of the Component. 

So I would write it as follows: 
```ts
const getUpperLeftIcon = (config: MediaPlayerTileConfig): JSX.Element | null => {
    if (config.typeName !== "local" && config.typeName !== "remote") return null;
    if (showDisabledIcon(config.audio)) return <DisabledMicIcon isLoading={isLoading(config.audio)} />;
    if (config.isSpeaking) return <SoundIcon />;
    return null;
};
```